nextcord/application_command.py

```diff
@@ -2038,6 +2038,7 @@ def user_command(
 ):
     """Creates a User context command from the decorated function.
     Used inside :class:`ClientCog`'s or something that subclasses it.
+
```

Comment:
maybe it fixes it but im not really sure.

Reply:
*it refers to `/home/docs/checkouts/readthedocs.org/user_builds/nextcord/checkouts/latest/nextcord/ext/commands/bot.py:docstring of nextcord.ext.commands.bot.BotBase.application_command_before_invoke:10: WARNING: Literal block ends without a blank line; unexpected unindent.` error.

Reply:
"not really sure": you can build docs locally